### PR TITLE
Use RVV spec 0.7.1 for C906.

### PR DESCRIPTION
**Pull request by xianyi**
Use RVV spec 0.7.1 for C906.

Change the compile flags in C906 toolchian file.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=2868) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=2868) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=2868) it.</sub>

**Comment by nihui** [comment]
depend on riscv toolchain updates https://github.com/Tencent/ncnn/pull/2872

File: src/layer/riscv/clip_riscv.cpp [hunk 1]
```diff
@@ -38,10 +38,18 @@ int Clip_riscv::forward_inplace(Mat& bottom_top_blob, const Option& opt) const
...
+#else
+	    "vle32.v    v0, (%0)        \n"
+#endif
             "vfmax.vf   v0, v0, %4      \n"
             "vfmin.vf   v0, v0, %5      \n"
+#ifdef RVV_SPEC_0_7
+	    "vse.v      v0, (%0)        \n"
+#else
             "vse32.v    v0, (%0)        \n"
+#endif
```
Comment (nihui) [inline src/layer/riscv/clip_riscv.cpp:52]: please fix the coding style
use space for indentation

**Review by nihui — CHANGES_REQUESTED** [review]

**Review by BUG1989 — APPROVED** [review]
LGTM!

**Review by nihui — APPROVED** [review]

**Comment by nihui** [comment]
thanks for your contribution!

**Pull request merged**